halyard-web/etc/init/halyard.conf

```diff
@@ -4,6 +4,4 @@ start on filesystem or runlevel [2345]
 
 expect fork
 
-stop on stopping spinnaker
-
-exec sudo -u spinnaker -g spinnaker /opt/halyard/bin/halyard 2>&1 > /var/log/spinnaker/halyard/halyard.log &
+exec sudo /opt/halyard/bin/halyard 2>&1 > /var/log/spinnaker/halyard/halyard.log &
```

Comment:
Because if halyard stops spinnaker, we don't want halyard to stop working.

It requires `sudo` to edit the logs, they're still readable for me.